step-controller-plugins/step-controller-plugins-timeseries/src/main/java/step/plugins/timeseries/TimeSeriesControllerPlugin.java

```diff
@@ -133,7 +140,15 @@ public void initializeData(GlobalContext context) throws Exception {
 		WebApplicationConfigurationManager configurationManager = context.require(WebApplicationConfigurationManager.class);
         configurationManager.registerHook(s -> Map.of(PARAM_KEY_EXECUTION_DASHBOARD_ID, newExecutionDashboard.getId().toString()));
 		configurationManager.registerHook(s -> Map.of(PARAM_KEY_ANALYTICS_DASHBOARD_ID, newAnalyticsDashboard.getId().toString()));
-		
+		AsyncTaskManager asyncTaskManager = context.require(AsyncTaskManager.class);
+		initTimeSeriesCollectionsData(asyncTaskManager);
+	}
+
+	private void initTimeSeriesCollectionsData(AsyncTaskManager asyncTaskManager) {
+		asyncTaskManager.scheduleAsyncTask((empty) -> {
```

Comment:
I assume this can take a while. Not sure what happens if the controller is stopped in between. We should at least log a clear message when this starts and ends